rust/src/proofs/api.rs

```diff
@@ -1408,6 +1415,432 @@ pub unsafe extern "C" fn fil_generate_single_window_post_with_vanilla(
     })
 }
 
+/// TODO: document
```

Comment:
I find it better not to put doc comments that contain only a TODO. This way tools can report if docs are missing..